bin/codemods/src/no-implicit-import-paths.js

```diff
@@ -0,0 +1,65 @@
+/** @format */
+/**
+ * Replaces import paths using implicit top-level
+ * directory structure with explicit import paths
+ *
+ * @example
+ * // input
+ * import Something from 'components/something'
+ *
+ * // output
+ * import Something from 'client/components/something'
+ */
+
+const chalk = require( 'chalk' );
+const fs = require( 'fs' );
+const path = require( 'path' );
+const config = require( './config' );
+
+const inClient = module => {
+	const hasIt = [ '', '.js', '.jsx' ].reduce( ( hasIt, suffix ) => {
+		if ( hasIt ) {
+			return hasIt;
+		}
+
+		try {
+			fs.accessSync( path.join( __dirname, '../../../client/', module + suffix ) );
+			return true;
+		} catch ( e ) {
+			return false;
+		}
+	}, false );
+
+	if ( ! hasIt ) {
+		console.log( chalk.yellow( `Not in client: ${ module }` ) );
+	}
+
+	return hasIt;
+};
+
+export default function transformer( file, api ) {
+	const j = api.jscodeshift;
+
+	return j( file.source )
+		.find( j.ImportDeclaration )
+		.filter( dec => {
+			const path = dec.value.source.value;
+
+			const isDot = path.startsWith( '.' );
+			const hasSlash = path.includes( '/' );
+
+			return ! isDot && hasSlash && inClient( path );
+		} )
+		.replaceWith( node => {
+			return Object.assign(
+				j.importDeclaration(
+					node.value.specifiers,
+					j.literal( node.value.source.value.replace( /^/, 'client/' ) )
```

Comment:
TIL you can prepend by replacing the `/^/`! Thats pretty cool.

I wonder if it would read simpler as:
`j.literal( 'client/' + node.source.value )`

Reply:
done - I have no idea what I was thinking